index.html

```diff
@@ -11,6 +11,22 @@
 
 </head>
 
+<header>
+    <div class="logo">
+        <img src="img/lambda-black.png">
+    </div>
```

Comment:
It is best to try to match the design file and have the nav align with the rest of the content like you did with the header.  Good job getting styles applied to all the anchors.